### PR TITLE
Add logging for common kvmfr initialization errors

### PR DESCRIPTION
**Pull request by MakiseKurisu**
`kvmfr` currently is not working correctly in my Linux VM so those logging messages are added to help diagnose the issue.

### CODE REVIEW

File: module/kvmfr.c [hunk 1]
```diff
@@ -370,7 +380,10 @@ static int kvmfr_pci_probe(struct pci_dev *dev, const struct pci_device_id *id)
   kdev->pDev  = device_create(kvmfr->pClass, NULL, kdev->devNo, NULL,
       KVMFR_DEV_NAME "%d", kdev->minor);
   if (IS_ERR(kdev->pDev))
+  {
+    printk(KERN_INFO "kvmfr: kvmfr_pci_probe: failed to create /dev/%s%d device!\n", KVMFR_DEV_NAME, kdev->minor);
```
Comment (gnif) [inline module/kvmfr.c:384]: Please limit lines to 80 columns

**Comment by gnif** [comment]
Looks good except for basic formatting and you need to add yourself to the AUTHORS file.

**Comment by gnif** [comment]
I am sorry but the email address in the AUTHORS file must be a valid real address you can be contacted at.

**Comment by MakiseKurisu** [comment]
I'll fix it, but there are quite a few people with GitHub noreply emails.

**Comment by gnif** [comment]
> I'll fix it, but there are quite a few people with GitHub noreply emails.

You are correct, however the AUTHORS file and it's requirements were added after these people had already contributed, as such they were added using only the details we have available.

**Pull request merged**